R/response.R

```diff
@@ -146,16 +160,29 @@ render_htmltools <- function(x) {
   href_deps <- grep("http", strsplit(rendered_deps, "\n")[[1]], value = TRUE)
   href_deps <- paste0(href_deps, collapse = "\n")
 
-  # add encoding and dependencies
-  q$closest("html")$find("head")$append(htmltools::tags$meta(charset = "UTF-8"))
-  q$closest("html")$find("head")$append(htmltools::HTML(href_deps))
-  q$closest("html")$find("head")$append(inline_deps)
+  # add encoding and dependencies for the first selected tag this avoid duplicates as append *appends* for each selected tag
+  q$closest("html")$find("head")$filter(function(x,i)i==1)$append(htmltools::tags$meta(charset = "UTF-8"))
```

Comment:
This may be a bit safer but I'm a bit confused about the idea that there may be multiple `<head>` tags generated. Should that issue be fixed upstream if it is the case. Have you noticed multiple `<head>`?